src/SprykerSdk/Acp/Console/AppTranslationCreateConsole.php

```diff
@@ -137,4 +382,89 @@ protected function askForConfirmation(
     ): string {
         return $this->getHelper('question')->ask($input, $output, new ChoiceQuestion($questionText, ['Yes', 'No'], 0));
     }
+
+    /**
+     * @return void
+     */
+    protected function writeLeftProcessMessage(): void
+    {
+        $this->output->writeln('<comment>Left the process, will continue with next steps.</comment>');
+    }
+
+    /**
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     *
+     * @return void
+     */
+    protected function printWelcomeMessage(OutputInterface $output): void
+    {
+        // TODO: complete the welcome message
+        $output->writeln('Welcome to the Translation Creator');
+    }
+
+    /**
+     * @param string $manifestFolder
+     * @param string $configurationFilePath
+     * @param string $translationFilePath
+     *
+     * @return \Transfer\ManifestCollectionTransfer
+     */
+    protected function getManifestCollection(
+        string $manifestFolder,
+        string $configurationFilePath,
+        string $translationFilePath
+    ): ManifestCollectionTransfer {
+        $manifestConditionsTransfer = (new ManifestConditionsTransfer())
+            ->setManifestFolder($manifestFolder)
+            ->setConfigurationFilePath($configurationFilePath)
+            ->setTranslationFilePath($translationFilePath);
+
+        $manifestCriteriaTransfer = (new ManifestCriteriaTransfer())
+            ->setManifestConditions($manifestConditionsTransfer);
+
+        return $this->getFacade()->getManifestCollection($manifestCriteriaTransfer);
+    }
+
+    /**
+     * @param \Transfer\ManifestCollectionTransfer $manifestCollectionTransfer
+     *
+     * @return array<string>
+     */
+    protected function getExistingLocales(ManifestCollectionTransfer $manifestCollectionTransfer): array
+    {
+        $locales = [];
+
+        foreach ($manifestCollectionTransfer->getManifests() as $manifestTransfer) {
+            $locales[] = $manifestTransfer->getLocaleNameOrFail();
+        }
+
+        return $locales;
+    }
+
+    /**
+     * @param \Transfer\ManifestCollectionTransfer $manifestCollectionTransfer
+     *
+     * @return array<string, array<string, string>>
+     */
+    protected function getExistingTranslations(ManifestCollectionTransfer $manifestCollectionTransfer): array
+    {
+        $translations = $manifestCollectionTransfer->getTranslation();
+
+        return $translations ? $translations->getTranslations() : [];
+    }
+
+    /**
+     * @param \Symfony\Component\Console\Input\InputInterface $input
+     * @param array $translations
+     *
+     * @return \Transfer\AppTranslationResponseTransfer
+     */
+    protected function saveTranslations(InputInterface $input, array $translations): AppTranslationResponseTransfer
```

Comment:
there are too many functions and logic in this command, could you move it to another layer